pkg/model/defaults/volumes_test.go

```diff
@@ -0,0 +1,54 @@
+/*
+Copyright 2019 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package defaults
+
+import (
+	"testing"
+
+	"k8s.io/kops/pkg/apis/kops"
+)
+
+func TestDefaultInstanceGroupVolumeSize(t *testing.T) {
```

Comment:
This test feels like it creates maintenance work rather than unit testing "kops logic". That is to say, `DefaultInstanceGroupVolumeSize` basically just returns different constants with a switch. 

IMO, changes that would break this test are likely to be intentional changes, so my vote would be to remove `volumes_test.go`.

Reply:
Fine.